lib/screens/v3_api.ex

```diff
@@ -39,7 +42,10 @@ defmodule Screens.V3Api do
         :not_modified
 
       {:response_success, %{status_code: status_code}} = response ->
-        log_api_error({:bad_response_code, response}, status_code: status_code)
+        {:bad_response_code, _} =
+          log_api_error({:bad_response_code, response}, status_code: status_code)
+
+        :bad_response_code
```

Comment:
This adds a new potential return value to the function if all retries fail, which makes me a liiiittle nervous, but I looked through our calls to `get_json` and they all wrap it in a `case` with a `_ ->` fallback, so we should be ok.

If there's any place where the function is called that doesn't have a catch-all fallback, it should also be easy to detect and fix as Sentry will report a `MatchError` if/when it receives `:bad_response_code`.

(Just laying out my thinking as to why this is change is safe)

Reply:
For peace of mind, I double-checked all references of this function and every references uses a `_ ->` fallback like you said. So I think we are good here. `log_api_error` already returns a few different values and none of those values are used for anything once the log has been captured.